tests/trainer/test_lr_finder.py

```diff
@@ -154,7 +154,7 @@ def test_accumulation_and_early_stopping(tmpdir):
         'Learning rate was not altered after running learning rate finder'
     assert len(lrfinder.results['lr']) == 100, \
         'Early stopping for learning rate finder did not work'
-    assert lrfinder._total_batch_idx == 190, \
+    assert lrfinder._total_batch_idx == 100 * 2, \
         'Accumulation parameter did not work'
```

Comment:
```suggestion
    assert lrfinder._total_batch_idx == 100 * 2, \
        'Accumulation parameter did not work'
    assert trainer.total_batch_idx == 0
```
What do you think? Should we also add this?

Reply:
Does this essentially ensure that the trainer never starts to train the model?

Reply:
yes, since there is accumulation happening in trainer and lr_finder, I would just include this assert for clarity.